src/constants.py

```diff
@@ -35,3 +35,9 @@
 }
 
 DATETIME_OLDEST = datetime.datetime(year=1970, month=1, day=1)
+
+ERROR_MSG_ANY_OF_THE_CATALOGUE_POST_MANDATORY_FIELDS_EMPTY = (
```

Comment:
This can be just a simple variable in the app.py file rather than a config.py file entry.

Reply:
moved the variable to the app.py file